### PR TITLE
add signature algorithm check.

### PR DESCRIPTION
**Pull request by s-tajima**

### CODE REVIEW

**Comment by s-tajima** [comment]
とりあえず書いてみたけどすずけんの流儀に合ってなかったら言って下さい

File: ats.go [hunk 1]
```diff
@@ -29,20 +30,48 @@ var acceptableCipher = []uint16{
...
+	// x509.DSAWithSHA256,
+	// x509.ECDSAWithSHA1,
+	x509.ECDSAWithSHA256,
+	x509.ECDSAWithSHA384,
+	x509.ECDSAWithSHA512,
+}
+
 // check verify ciphers and TLS version
 func check(resp *http.Response) (bool, error) {
+	var ciperCheck = false
```
Comment (suzuken) [inline ats.go]: should `cipherCheck`

**Comment by suzuken** [comment]
@s-tajima Thank you for your pull requests. I added some comments.
- If you can, please add test for testing signature.
- Some typo exists. Please fix it.

**Comment by s-tajima** [comment]
fixed typo.
I will add test later.

**Pull request merged**